public-ui/src/app/views/pm/pm.component.html

```diff
@@ -25,9 +25,9 @@ <h1>{{title}}</h1>
               (click)="showMeasurement(selectedGroup, c)" class="btn btn-link">{{c.conceptName}} </button>
           </div>
         </div>
+        <div *ngIf="selectedConcept && (selectedConcept.conceptId === '903111' || selectedConcept.conceptId === '903120')" class="participant-count">Participant count: {{selectedConcept.analyses.countAnalysis.results[0].countValue}}</div>
```

Comment:
Taking it to the top so that it does not interfere in the graph space.

Reply:
can you pipe the value so it has a comma after the thousands?

Reply:
Done. Thanks.